Sources/Client/Views/Config/SettingsView.swift

```diff
@@ -1,27 +1,58 @@
 import SwiftUI
 import DeltaCore
 
+enum SettingsState {
+  case none, accounts, plugins, update, troubleshooting, video
+}
+
 struct SettingsView: View {
   var isInGame: Bool
   var eventBus: EventBus?
   var done: () -> Void
   
-  init(isInGame: Bool, eventBus: EventBus?, onDone done: @escaping () -> Void) {
+  /// The `NavigationLink` to be selected on initializaiton
+  @State private var initialLandingPage: SettingsState? = nil
+  
+  init(isInGame: Bool,
+       eventBus: EventBus?,
+       landingPage: SettingsState? = nil,
+       onDone done: @escaping () -> Void) {
     self.isInGame = isInGame
     self.eventBus = eventBus
     self.done = done
+    self._initialLandingPage = State(initialValue: landingPage)
   }
   
   var body: some View {
     NavigationView {
       List {
         if !isInGame {
-          NavigationLink("Accounts", destination: AccountSettingsView().padding())
-          NavigationLink("Update", destination: UpdateView().padding())
-					NavigationLink("Plugins", destination: PluginView().padding())
+          NavigationLink(
+            "Accounts",
+            destination: AccountSettingsView().padding(),
+            tag: SettingsState.accounts,
+            selection: $initialLandingPage)
+          NavigationLink(
+            "Update",
+            destination: UpdateView().padding(),
+            tag: SettingsState.update,
+            selection: $initialLandingPage)
+          NavigationLink(
+            "Troubleshooting",
+            destination: TroubleShootingView(),
+            tag: SettingsState.troubleshooting,
+            selection: $initialLandingPage)
+	    NavigationLink(
```

Comment:
Indentation seems to have gone a little funky here

Reply:
Yea, same as here https://github.com/stackotter/delta-client/pull/41#discussion_r744137062